ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -6130,6 +6130,155 @@ spec:
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting specifies whether to require the use of a specific security layer to secure communications between clients and RD Session Host servers during Remote Desktop Protocol (RDP) connections.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Enabled: SSL':
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Remote Desktop Services\Remote Desktop Session Host\Security\Require use of specific security layer for remote (RDP) connections'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal Services\\SecurityLayer' AND data = 2);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_18.9.65.3.9.3
+  contributors: artemist-work
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Require user authentication for remote connections by using Network Level Authentication' is set to 'Enabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting allows you to specify whether to require user authentication for remote connections to the RD Session Host server by using Network Level Authentication.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Enabled':
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Remote Desktop Services\Remote Desktop Session Host\Security\Require user authentication for remote connections by using Network Level Authentication'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal Services\\UserAuthentication' AND data = 1);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_18.9.65.3.9.4
+  contributors: artemist-work
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Set client connection encryption level' is set to 'Enabled: High Level'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting specifies whether to require the use of a specific encryption level to secure communications between client computers and RD Session Host servers during Remote Desktop Protocol (RDP) connections.
+    This policy only applies when you are using native RDP encryption.
+    However, native RDP encryption (as opposed to SSL encryption) is not recommended.
+    This policy does not apply to SSL encryption.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Enabled', then 'High Level':
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Remote Desktop Services\Remote Desktop Session Host\Security\Set client connection encryption level'
+  query: |
+    TODO
```

Comment:
This query should be moved to the NON-completed yml if not ready, otherwise just remove the TODO keyword here

Reply:
Oops, that was a mistake, this query works.

Reply:
excellent, thanks for addressing it! I was thinking that it was most likely a leftover from a previous query